### PR TITLE
4.12 monitoring_alerts updates-2

### PR DESCRIPTION
**Pull request by Tai-RedHat**
fix 4.12 error in OCP-21150/OCP-29300
https://issues.redhat.com/browse/OCPQE-12648

runner job:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/658650/console

### CODE REVIEW

File: lib/rules/web/admin_console/4.12/monitoring_alerts.xyaml [hunk 1]
```diff
@@ -191,13 +191,13 @@ set_silence_duration:
...
     selector:
-      xpath: //ul[@class='pf-c-dropdown__menu']//button[text()='<duration>']
+      xpath: //label[text()='For...']//..//button
     op: click
 set_silence_duration_null:
   action: click_duration_dropdown_button
   element:
     selector:
-      xpath: //ul[@class='pf-c-dropdown__menu']//button[text()='-']
+      xpath: //ul[@class='pf-c-dropdown__menu']//li[1]
```
Comment (lihongyan1) [inline lib/rules/web/admin_console/4.12/monitoring_alerts.xyaml:200]: Better change as `//ul[@class='pf-c-dropdown__menu']//a[text()='-']`

**Comment by Tai-RedHat** [comment]
passed runner job:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/659488/console
```
11-01 15:03:16.242  2 scenarios (2 passed)
11-01 15:03:16.242  42 steps (42 passed)
11-01 15:03:16.242  3m42.104s
...
11-01 15:03:16.939  Finished: SUCCESS
```

**Comment by Tai-RedHat** [comment]
@yapei please help to approve when you are available, thanks.

**Pull request merged**